starter_code/src/movies.js

```diff
@@ -1,20 +1,59 @@
 /* eslint no-restricted-globals: 'off' */
-// Turn duration of the movies from hours to minutes 
-
-
-// Get the average of all rates with 2 decimals 
-
-
+// Turn duration of the movies from hours to minutes
+function stringToMinutes(string) {
+  var arr = string.split("h");
+  if (arr.length < 2) {
+    arr.unshift("0");
+  }
+  var hours = Number(arr[0]);
+  var minutes = Number(arr[1].split("min")[0]);
+
+  return hours * 60 + minutes;
+}
+
+function turnHoursToMinutes(array) {
+  var newArray = [];
+  var result = array.map(function(movie) {
+    var obj = {
+      title: movie.title,
+      year: movie.year,
+      director: movie.director,
+      duration: stringToMinutes(movie.duration),
+      genre: movie.genre,
+      rate: movie.rate
+    };
+    newArray.push(obj);
```

Comment:
you should simply return `result` from the `turnHoursToMinutes` function, no need to push to a `newArray` :)